tests/tests_full_scan_engine_udf.py

```diff
@@ -559,13 +560,20 @@ async def test_udf_postgresql_and_mysql(self):
         # Run the query on the aidb database
         logger.info(f'Running query {aidb_query} in aidb database')
         aidb_res = aidb_engine.execute(aidb_query)
-        # # TODO: may have problem with decimal number
         assert len(gt_res) == len(aidb_res)
-        # assert Counter(gt_res) == Counter(aidb_res)
+
+        gt_res = sorted(gt_res)
+        aidb_res = sorted(aidb_res)
+
+        for i in range(len(gt_res)):
+          for element1, element2 in zip(gt_res[i], aidb_res[i]):
```

Comment:
Does it work if the outcome is limited to a single element? For instance, in the SQL query 'select x_min from objects00,' does it return (1, 2, 3), or ((1), (2), (3))? If it's the former, this could lead to errors.

Reply:
the result will always be a list of tuples, even if it just has one element